maat-court-data-api/src/test/java/gov/uk/courtdata/users/controller/UserSummaryControllerTest.java

```diff
@@ -33,13 +35,79 @@ void givenIncorrectParameters_whenUpdateCCOutcomeIsInvoked_thenErrorIsThrown() t
 
     @Test
     void givenAValidParameters_whenGetUserSummaryIsInvoked_thenReturnStatusOK() throws Exception {
-        when(userSummaryService.getUserSummary(TestModelDataBuilder.TEST_USER))
+        when(userSummaryService.getUserSummary(TEST_USER))
                 .thenReturn(TestModelDataBuilder.getUserSummaryDTO());
-        mvc.perform(MockMvcRequestBuilders.get(ENDPOINT_URL + TestModelDataBuilder.TEST_USER))
+        mvc.perform(MockMvcRequestBuilders.get(ENDPOINT_URL + TEST_USER))
+                .andExpect(status().isOk())
+                .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+                .andExpect(jsonPath("$.username").value(TEST_USER));
+    }
+
+    @Test
+    void givenIncorrectParameters_whenCreateUserIsInvoked_thenErrorIsThrown() throws Exception {
+        mvc.perform(MockMvcRequestBuilders.post(USER_ENDPOINT_URL)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().is4xxClientError());
+    }
+
+    @Test
+    void givenValidParameters_whenCreateUserIsInvoked_thenReturnStatusOK() throws Exception {
+        String requestJson = """
+                {
+                  "username" : "test-f",
+                  "loggedIn" : "Y",
+                  "currentSession" : "mock-session"
+                }
+                """;
+
+        mvc.perform(MockMvcRequestBuilders.post(USER_ENDPOINT_URL).content(requestJson)
+                        .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk());
+    }
+
+    @Test
+    void givenValidParameters_whenGetUserIsInvoked_thenReturnStatusOK() throws Exception {
+        when(userSummaryService.getUser(TEST_USER))
+                .thenReturn(UserEntity.builder().username(TEST_USER).build());
+        mvc.perform(MockMvcRequestBuilders.get(USER_ENDPOINT_URL + TEST_USER))
                 .andExpect(status().isOk())
                 .andExpect(content().contentType(MediaType.APPLICATION_JSON))
-                .andExpect(jsonPath("$.username").value(TestModelDataBuilder.TEST_USER));
+                .andExpect(jsonPath("$.username").value(TEST_USER));
     }
 
+    @Test
+    void givenIncorrectParameters_whenPatchUserIsInvoked_thenErrorIsThrown() throws Exception {
+        mvc.perform(MockMvcRequestBuilders.post(USER_ENDPOINT_URL)
+                        .contentType(MediaType.APPLICATION_JSON))
```

Comment:
For patch, you need to change to MockMvcRequestBuilders.patch

Reply:
Fixed.